components/Header/Header.tsx

```diff
@@ -10,12 +10,13 @@ interface HeaderProps {
  * Displays the title and subtitle
  */
 const Header: React.FC<HeaderProps> = ({ title, subtitle }) => (
-  <div>
+  <div className="knowledge-panel-title">
     <span>
       {title}
     </span>
+    {'\n'}

```

Comment:
```suggestion
```
Using new lines isn't good practice - use CSS instead. Specifically, I would add a bottom margin to the title, or a top margin to the subtitle, probably 4 px or 8 px.

Also, if the "proper" way to do it would be `<br>` just btw